packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -39,14 +38,13 @@ import 'package:amplify_authenticator/src/theme/amplify_theme.dart';
 import 'package:amplify_authenticator/src/widgets/authenticator_banner.dart';
 import 'package:amplify_authenticator/src/widgets/form.dart';
 import 'package:amplify_core/amplify_core.dart';
-import 'package:amplify_flutter/src/config/amplify_config.dart';
+import 'package:amplify_flutter/amplify.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
 export 'package:amplify_auth_cognito/amplify_auth_cognito.dart'
     show AuthProvider;
-export 'package:amplify_flutter/src/config/auth/password_policy_characters.dart';
-export 'package:amplify_flutter/src/config/auth/password_protection_settings.dart';
+export 'package:amplify_flutter/amplify.dart';
```

Comment:
Yeah, I think so. Why should they need to import two packages?

Reply:
Okay. I think it is fine. Mainly wanted to make sure it was intentional.

You could make that same argument with amplify_auth_cognito as well. Right now only AuthProvider is exported here from amplify_auth_cognito. But I guess that is a little different since you don't have to use anything from amplify_auth_cognito to use the authenticator.